### ISSUE TITLE
Fold/unfold don't work with decorated functions

### ISSUE THREAD

**Issue by codecat555** [pokey/cursorless-vscode#429]
I've noticed that the fold and unfold commands don't seem to work when a function has been decorated, e.g.

`# taken from https://talon.wiki/unofficial_talon_docs/#captures
@mod.capture(rule="all | ((north | south) [(east | west)] | east | west)")
def compass_direction(m) -> Direction:`

I searched for existing issues and for settings that might affect this but didn't find anything.

**Comment by MartinRykfors** [comment]
I ran into this problem while working on a big class and I wanted to `fold every funk` in order to get a better overview. All functions with a decorator remain unfolded and the entire class gets folded.

Andreas suggested using `fold every function name` which seems to be a fine workaround if anyone else also has this problem.

**Comment by pokey** [comment]
Yeah good call. Should work as a workaround for folding a single function as well, eg just "fold funk name"
